apitest/docs/api-beta-test-guide.md

```diff
@@ -0,0 +1,492 @@
+putting up a 15% security deposit, the `createoffer` command to do that would be:
+```
+$ ./bisq-cli --password=xyz --port=9998 createoffer
+    --payment-account=f3c1ec8b-9761-458d-b13d-9039c6892413
+    --direction=SELL
+    --currency-code=JPY
+    --amount=0.125
+    --market-price-margin=0.5
+    --security-deposit=15.0
+    --fee-currency=BSQ
+```
+
+The `trade-simulation.sh` script options that would generate the previous `createoffer` example is:
+```
+$ apitest/scripts/trade-simulation.sh -d sell -c jp -m 0.5  -a 0.125
+```
+
+### Browsing Your Own Offers
+
+There are different commands to browse available offers you can take, and offers you created.
+
+To see all offers you created with a specific direction (BUY|SELL) and currency (CAD|EUR|USD|...):
+```
+$ ./bisq-cli --password=xyz --port=9998 getmyoffers --direction=<BUY|SELL> --currency-code=<currency-code>
+```
+
+To look at a specific offer you created:
+```
+$ ./bisq-cli --password=xyz --port=9998 getmyoffer --offer-id=<offer-id>
+```
+
+### Browsing Available Offers
+
+To see all available offers you can take, with a specific direction (BUY|SELL) and currency (CAD|EUR|USD|...):
+```
+$ ./bisq-cli --password=xyz --port=9998 getoffers --direction=<BUY|SELL> --currency-code=<currency-code>
+```
+
+To look at a specific, available offer you could take:
+```
+$ ./bisq-cli --password=xyz --port=9998 getoffer --offer-id=<offer-id>
+```
+
+### Removing An Offer
+
+To cancel one of your offers:
+```
+$ ./bisq-cli --password=xyz --port=9998 canceloffer --offer-id=<offer-id>
+```
+The offer will be removed from other Bisq users' offer views, and paid transaction fees will be forfeited.
+
+### Editing an Existing Offer
+
+Editing existing offers is not yet supported.  You can cancel and re-create an offer, but paid transaction fees
+for the canceled offer will be forfeited.
+
+### Taking Offers
+
+Taking an available offer involves two CLI commands: `getoffers` and `takeoffer`.
+
+A CLI user browses available offers with the getoffers command.  For example, the user browses SELL / EUR offers:
+```
+$ ./bisq-cli --password=xyz --port=9998  getoffers --direction=SELL --currency-code=EUR
+```
+
+And takes one of the available offers with an EUR payment account ( id `fe20cdbd-22be-4b8a-a4b6-d2608ff09d6e`)
+with the `takeoffer` command:
+```
+$ ./bisq-cli --password=xyz --port=9998 takeoffer
+    --offer-id=83e8b2e2-51b6-4f39-a748-3ebd29c22aea
+    --payment-account=fe20cdbd-22be-4b8a-a4b6-d2608ff09d6e
+    --fee-currency=btc
+```
+The taken offer will be used to create a trade contract.  The next section describes how to use the Api to execute
+the trade.
+
+### Completing Trade Protocol
+
+The first step in the Bisq trade protocol is completed when a `takeoffer` command successfully creates a new trade from
+the taken offer.  After the Bisq servers prepare the trade, its status can be viewed with the `gettrade` command:
```

Comment:
```suggestion
the taken offer.  After the Bisq nodes prepare the trade, its status can be viewed with the `gettrade` command:
```

I see how 'server' might make sense in this context but 'node' is more commonly-used throughout docs.